Libraries/Components/Keyboard/KeyboardAvoidingView.js

```diff
@@ -43,15 +43,19 @@ type KeyboardChangeEvent = {
 const viewRef = 'VIEW';
 
 /**
- * It is a component to solve the common problem of views that need to move out of the way of the virtual keyboard.
- * It can automatically adjust either its position or bottom padding based on the position of the keyboard.
+ * This is a component to solve the common problem of views that need to move out of the way of the virtual keyboard.
+ * It can automatically adjust either its height, position or bottom padding based on the position of the keyboard.
  */
 const KeyboardAvoidingView = createReactClass({
   displayName: 'KeyboardAvoidingView',
   mixins: [TimerMixin],
 
   propTypes: {
     ...ViewPropTypes,
+    /**
+     * Specify how the `KeyboardAvoidingView` will react to the presence of
+     * the keyboard. Will it adjust its height, position or bottom padding?
```

Comment:
Can you phrase this not as a question. Something like "Specify if it will adjust ..."